webview/src/experiments/components/table/CellHintTooltip.tsx

```diff
@@ -1,19 +1,26 @@
 import React, { ReactNode, ReactElement } from 'react'
+import { TippyProps } from '@tippyjs/react'
 import styles from './styles.module.scss'
 import Tooltip, {
   CELL_TOOLTIP_DELAY
 } from '../../../shared/components/tooltip/Tooltip'
 
-export const CellHintTooltip: React.FC<{
+export type CellHintTooltipProps = {
   tooltipContent: ReactNode
   children: ReactElement
-}> = ({ tooltipContent, children }) => {
+}
+
+export const CellHintTooltip: React.FC<CellHintTooltipProps & TippyProps> = ({
+  tooltipContent,
+  children,
+  delay = [CELL_TOOLTIP_DELAY, 0]
```

Comment:
What does the second delay mean?

Reply:
How long it takes for the [tooltip to hide](https://atomiks.github.io/tippyjs/v6/all-props/#delay) :)